hooks/checkout

```diff
@@ -90,6 +90,9 @@ checkout() {
   git reset --hard
   git clean -ffxdq
 
+  git_log=$(mktemp --suffix .log --tmpdir githublog.XXXXXXXXXX)
+  trap 'buildkite-agent artifact upload "${git_log}";rm -rf "${git_log}"' EXIT
```

Comment:
You can't add a second trap, there's already one on line 78, so one of the two traps will get overwritten by the other: https://github.com/arromer/github-fetch-buildkite-plugin/blob/94bb24b24c34ebf703f861f5a8bb17b1d646ed0d/hooks/checkout#L78